app/screens/home/search/results/file_results.tsx

```diff
@@ -48,121 +46,98 @@ const FileResults = ({
     canDownloadFiles,
     fileChannels,
     fileInfos,
-    publicLinkEnabled,
     paddingTop,
+    publicLinkEnabled,
     searchValue,
 }: Props) => {
     const theme = useTheme();
-    const isTablet = useIsTablet();
     const insets = useSafeAreaInsets();
-    const [lastViewedIndex, setLastViewedIndex] = useState<number | undefined>(undefined);
-    const containerStyle = useMemo(() => ({top: fileInfos.length ? 8 : 0}), [fileInfos]);
+    const isTablet = useIsTablet();
+
+    const [optionsOpen, setOptionsOpen] = useState(false);
```

Comment:
On tablet, this seems to be set back to false properly, but I don't see the logic to properly set to false on mobile. Maybe I am missing something.

Reply:
`setOptionsOpen` is passed to `FileResult` as a prop and gets called by the following functions:

* `handleOptionsPress` 
    * options button is pressed (on mobile or tablet)
    * tells `FileResults` the options menu is open
* `handleSetAction` 
  * after the toast is displayed which will set the `action` value to something other than `none` 
  * tells `FileResults` the options menu is closed

<img width="602" alt="image" src="https://user-images.githubusercontent.com/7575921/193859815-c6e4678e-5f1c-4dc9-b979-751bd5fc9005.png">

Does that make sense?

Reply:
Is it somehow called when we close the bottom sheet on mobile (without clicking on any of the options and therefore not calling the handleSetAction)?

Reply:
ugh!!  Yes, this is still a problem.  What I really need is a way to detect if the bottomSheet is a labeled or has a key, ie. (the bottom sheet is open and it is the `fileOptions` bottomsheet.)

Reply:
This corner case is getting more complicated that I like.  I removed the related code and created a ticket to follow up after beta release.  https://mattermost.atlassian.net/browse/MM-47365